CHANGELOG.md

```diff
@@ -2,7 +2,7 @@
 
 ## [next]
 
-## 0.19.1-beta
+## 0.19.1
```

Comment:
Put this on next, as we will merge other PRs before releasing 0.19.1

Reply:
ok